wallet-api/src/Wallet/Emulator/Client.hs

```diff
@@ -16,15 +35,41 @@ api = Proxy
 wallets :: ClientM [Wallet]
 fetchWallet :: Wallet -> ClientM Wallet
 createWallet :: Wallet -> ClientM NoContent
-createPaymentWithChange :: Wallet -> Value -> ClientM (Set TxIn', TxOut')
-payToPublicKey :: Wallet -> Value -> ClientM TxOut'
-submitTxn :: Wallet -> Tx -> ClientM ()
+myKeyPair' :: Wallet -> ClientM KeyPair
```

Comment:
I still think it's easier to just disambiguate these things with qualified imports, but w/e.